src/utilities/error.c

```diff
@@ -85,6 +85,66 @@ hypre_error_handler(const char *filename, HYPRE_Int line, HYPRE_Int ierr, const
    hypre__global_error = err;
 }
 
+/*--------------------------------------------------------------------------
+ *--------------------------------------------------------------------------*/
+
+void
+hypre_error_handler_rank_0(HYPRE_Int   myid,
```

Comment:
Remove this function.  I don't understand its purpose and you aren't using it anyway.

Reply:
Done in [8ed50af](https://github.com/hypre-space/hypre/pull/1040/commits/8ed50af2b09c7125308badda38a3f57b4c1171ba)